apps/scan/src/main/scala/org/lfdecentralizedtrust/splice/scan/admin/api/client/BftScanConnection.scala

```diff
@@ -1080,57 +1177,210 @@ object BftScanConnection {
       httpClient: HttpClient,
       templateDecoder: TemplateJsonDecoder,
   ): Future[BftScanConnection] = {
+
     val builder = buildScanConnection(upgradesConfig, clock, retryProvider, loggerFactory)
+    val logger = loggerFactory.getTracedLogger(getClass)
+
     config match {
-      case BftScanClientConfig.TrustSingle(url, amuletRulesCacheTimeToLive) =>
-        // If this fails to connect, fail and let it retry
-        val connectionF = builder(url, amuletRulesCacheTimeToLive)
-        connectionF
-          .map(conn =>
-            new BftScanConnection(
-              spliceLedgerClient,
-              amuletRulesCacheTimeToLive,
-              new TrustSingle(conn, retryProvider, loggerFactory),
-              clock,
-              retryProvider,
-              loggerFactory,
-            )
-          )
-      case BftScanClientConfig.Bft(seedUrls, scansRefreshInterval, amuletRulesCacheTimeToLive) =>
+      case BftScanClientConfig.TrustSingle(url, ttl) =>
+        for {
+          conn <- builder(url, ttl)
+          scanList = new TrustSingle(conn, retryProvider, loggerFactory)
+        } yield new BftScanConnection(
+          spliceLedgerClient,
+          ttl,
+          scanList,
+          clock,
+          retryProvider,
+          loggerFactory,
+        )
+
+      case ts @ BftScanClientConfig.BftCustom(_, _, _, _, _) =>
+        /*
+         * The initialization for `BftCustom` is more complex than for `AllDsoScansBft`.
+         *
+         * `AllDsoScansBft` can bootstrap with a single seed URL because its BFT threshold is
+         * dynamic (f+1). On the first `refresh`, if it only knows about one node, the threshold becomes 1,
+         * allowing it to make a non-BFT discovery call to find the rest of the network.
+         *
+         * This does not work for `BftCustom` because its threshold is static and determined by the
```

Comment:
Sorry, should've made this a bit clearer, but for the initial discovery the previous bft was creating a bft connection from the seeds and had a threshold based on the number of seeds. What's stopping us from doing the same thing here?